tutormfe/patches/openedx-lms-development-settings

```diff
@@ -11,9 +11,41 @@ LEARNING_MICROFRONTEND_URL = "http://{{ MFE_HOST }}:{{ MFE_LEARNING_MFE_APP["por
 PROFILE_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https://{% else %}http://{% endif %}{{ MFE_HOST }}:{{ MFE_PROFILE_MFE_APP["port"] }}/{{ MFE_PROFILE_MFE_APP["name"] }}/u/"
 {% endif %}
 
+{% if MFE_DISCUSSIONS_MFE_APP %}
+DISCUSSIONS_MICROFRONTEND_URL = "http://{{ MFE_HOST }}:{{ MFE_DISCUSSIONS_MFE_APP["port"] }}/{{ MFE_DISCUSSIONS_MFE_APP["name"] }}"
+DISCUSSIONS_MFE_FEEDBACK_URL = None
+{% endif %}
+
 {% for app in iter_values_named(suffix="MFE_APP") %}
 # {{ app["name"] }} MFE
 CORS_ORIGIN_WHITELIST.append("http://{{ MFE_HOST }}:{{ app["port"] }}")
 LOGIN_REDIRECT_WHITELIST.append("{{ MFE_HOST }}:{{ app["port"] }}")
 CSRF_TRUSTED_ORIGINS.append("{{ MFE_HOST }}:{{ app["port"] }}")
 {% endfor %}
+
+MFE_CONFIG = {
```

Comment:
Configuration patches are copied wholesale from the [current implementation](https://github.com/overhangio/tutor-mfe/blob/master/tutormfe/patches/openedx-lms-production-settings)